tests/test_generators/test_pydanticgen.py

```diff
@@ -578,6 +578,59 @@ def test_pydantic_arrays():
     assert temperature_dataset.temperatures_in_K == temperatures
 
 
+def test_pydantic_cardinality():
+
+    unit_test_schema = """
+id: https://example.org/arrays
+name: arrays-cardinality-example
+title: Array Cardinality Example
+description: |-
+    Example LinkML schema to test array cardinality
+license: MIT
+
+default_prefix: example
+
+imports:
+    - linkml:types
+
+classes:
+    CardinalityArray:
+        description: A class with arrays of different cardinality constraints
+        attributes:
+            minimum_cardinality_array:
+                implements:
+                    - linkml:elements
```

Comment:
i don't think you need these in here but it also doesn't hurt :)

Reply:
I'll go ahead and remove it then, for the sake of clarity / concision